src/Core/AdminConsole/Entities/Organization.cs

```diff
@@ -100,19 +107,35 @@ public void SetNewId()
         }
     }
 
+    /// <summary>
+    /// Returns the name of the organization, decoded from HTML ready for display
+    /// </summary>
+    public string OrganizationName()
```

Comment:
nit: could we name this `DisplayName()`? Just so there's a little more differentiation from the `Name` property at a glance.